frontend/insurance/src/app/modules/edit-claim/edit-claim.component.ts

```diff
@@ -74,4 +74,23 @@ export class EditClaimComponent implements OnInit {
       },
     );
   }
+
+  editClaim(status: number = 0, increment: number = 0): void {
```

Comment:
Does increment need to be a parameter? We're always going to be increasing the claim by 1 and the user has no way to increase the claim by 2 statuses

Reply:
it's not always going to be incremented by 1, sometimes it'll be decremented since you can move back and forth between statuses.

Reply:
I agree with this. Is there anyway to just keep this in memory? That way you don't need to pass through either variables?